oas

```diff
@@ -490,11 +533,92 @@ components:
         archived: false
         name: 'Marta'
         public_identifier: 'DE 123456789'
-        farm_identifier: '674'
+        alternative_identifier: '674'
         pen_urn: 'urn:animal_service:FDBE6F55-6179-4B88-9177-4AFBED33ADAC'
-        date_birth: '2021-03-06T12:34:45Z'
         sex: 'f'
         species: 'cattle'
+
+    Event:
+      type: object
+      properties:
+        date:
+          type: string
+          format: date-time
+          description: date when this event happened
+        category:
+          $ref: '#/components/schemas/EventCategory'
+        animal_urn:
+          $ref: '#/components/schemas/URN'
+        payload: 
+          oneOf:
+            - description: birth event
+              type: object
+              properties:
+                mother_public_identifier:
+                  type: string
+                father_public_identifier:
+                  type: string
+            - description: arrival event
+              type: object
+              properties:
+                 arrival_reason:
+                  type: string
+                  description: reason why this animal arrives on the farm
+                  enum:
+                    - 'buyed'
```

Comment:
bought

Reply:
should there not be "born" as well?

Reply:
see the event above, there is birth event, which includes some more information like father and mother ids